thirdeye/thirdeye-frontend/app/pods/components/filter-bar/component.js

```diff
@@ -81,7 +83,10 @@ export default Ember.Component.extend({
       });
 
       // Now add new initialized props to block item
-      Object.assign(block, { filtersArray, isHidden });
```

Comment:
@ttbach this was generating an error, you'll need to use Ember.set instead of Object assignment